### PR TITLE
Add a couple variable overrides

### PR DESCRIPTION
**Pull request by halkeye**
1) Maven - was testing out things with maven 2.5 vs 2.6, so created an override to change it so didn't need (at the time) to change my path
2) Local checkout dir - Allows local checkout dir to be set without changing the makefile

### CODE REVIEW

**Comment by alecharp** [comment]
I don't think there is a need to put that in the project. This is more a customization of the run process to suit your need. I have the same kind of modification locally.

**Comment by halkeye** [comment]
> I don't think there is a need to put that in the project. This is more a customization of the run process to suit your need. I have the same kind of modification locally.

Just one less thing to have to change often

/shrug

File: Makefile [hunk 1]
```diff
@@ -69,10 +76,10 @@ demo-jdk11: plugins-compat-tester-cli/target/plugins-compat-tester-cli.jar tmp/j
 	     -reportFile $(CURDIR)/out/pct-report.xml \
 	     -failOnError \
 	     -workDirectory $(CURDIR)/work -skipTestCache true \
-	     -mvn $(shell which mvn) -war tmp/jenkins-war-$(JENKINS_VERSION).war \
+	     -mvn $(MVN) -war tmp/jenkins-war-$(JENKINS_VERSION).war \
 	     -testJDKHome $(TEST_JDK_HOME) \
 	     -testJavaArgs "-p $(CURDIR)/tmp/jaxb-api-$(JAXB_API_VERSION).jar:$(CURDIR)/tmp/javax.activation-$(JAF_VERSION).jar --add-modules java.xml.bind,java.activation -cp $(CURDIR)/tmp/jaxb-impl-$(JAXB_VERSION).jar:$(CURDIR)/tmp/jaxb-core-$(JAXB_VERSION).jar" \
-	     -includePlugins $(PLUGIN_NAME)
+			 $(LOCAL_CHECKOUT_DIR_CMD) -includePlugins $(PLUGIN_NAME)
```
Comment (oleg-nenashev) [inline Makefile]: Unrelated change

File: Makefile [hunk 2]
```diff
@@ -57,9 +64,9 @@ demo-jdk8: plugins-compat-tester-cli/target/plugins-compat-tester-cli.jar tmp/je
 	     -reportFile $(CURDIR)/out/pct-report.xml \
 	     -failOnError \
 	     -workDirectory $(CURDIR)/work -skipTestCache true \
-	     -mvn $(shell which mvn) -war tmp/jenkins-war-$(JENKINS_VERSION).war \
+	     -mvn $(MVN) -war tmp/jenkins-war-$(JENKINS_VERSION).war \
 	     -testJDKHome $(TEST_JDK_HOME) \
-	     -includePlugins $(PLUGIN_NAME)
+			 $(LOCAL_CHECKOUT_DIR_CMD) -includePlugins $(PLUGIN_NAME)
```
Comment (oleg-nenashev) [inline Makefile]: Unrelated change

**Comment by halkeye** [comment]
I'm just gonna close it
